react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -53,7 +52,11 @@ export default class JabraService extends VendorImplementation {
 
     deviceLabelMatchesVendor(label: string): boolean {
         const lowerLabel = label.toLowerCase();
```

Comment:
Does this `label.toLowerCase()` need to be falsy checked?

Reply:
in `headset.ts` where this function is called, it checks that the label is falsy there first. if it is, it clears out the selected implementation there if there is one. if not, then it will call this function. should that logic be moved around and in here instead? that probably makes more sense now that i think about it